content/_releases/22/14.md

```diff
@@ -0,0 +1,267 @@
+---
+title: Product Update - v22.14
+description: Details of the product update v22.14 from 11th of April 2022.
+layout: rel
+version: "22.14"
+section: Product Updates
+category: product-updates
+releaseDate: 2022-04-11
+redirect_from:
+  - /releases/22.14.html
+---
+
+## Improvements and Updates
+
+### Step designer: Sample data part
+
+To provide a unified experience during the flow step design, we improved the Sample
+data retrieval section. The look and feel is now more in line with our new
```

Comment:
Remove minimalist -  nested scroll bars is not minimalist :))

Reply:
Done